example-library/src/main/java/uk/gov/gchq/palisade/example/common/ExampleUser.java

```diff
@@ -46,6 +46,13 @@ public ExampleUser trainingCompleted(final TrainingCourse... trainingCompleted)
         return this;
     }
 
+    public ExampleUser trainingCompleted(final EnumSet<TrainingCourse> trainingCompleted) {
```

Comment:
these need updating for the new equals, hashcode, tostring, getter, setter and mutator format

Reply:
Updated

Reply:
Updated using the new generator templates